### ISSUE TITLE
Multisport Distance not showing up

### ISSUE THREAD

**Issue by evanodell** [jimmykane/quantified-self#221]
Noticed the problem in this upload. The full distance shows up on the map and other data is available, but the distance values are given as 0km for the 2nd running section: https://quantified-self.io/user/fRf6bCiiKcWGtwF8dXYuUwJgnGe2/event/knDuomAgpurrfTQWyw7M

I'm happy to figure out a solution but I'm not sure what the best place to start is.

**Comment by jimmykane** [comment]
It it possible to make the link public so I can take a look ?

**Comment by evanodell** [comment]
Ah didn't realise I had it private. Should work now.

**Comment by jimmykane** [comment]
I see the issue :-) 

I ll investigate and come back. 

I also see another issue. The map has a great period marked as paused. 


If you have a link from the original file (FIT) that would be great to check why so many pause/start events

**Comment by evanodell** [comment]
Here is the FIT file: [5e76296d33564d7a05716b96.fit.zip](https://github.com/jimmykane/quantified-self/files/5554590/5e76296d33564d7a05716b96.fit.zip)

**Comment by evanodell** [comment]
Probably had a lot of pause/start events because of autopause and me stopping a lot.

**Comment by jimmykane** [comment]
I see that apart from the fit file first session the other 2 don't have a total distance. Seeing that this is a Suunto, I ll file a bug with them. 

However, I can fix this on the lib side. QS uses sports lib. 

The fix is easy. 

At some point the lib generates distance from the samples. But I have explicitly asked for 0 distance to be ignored. I can change it to try also for distance of 0. 

Here: https://github.com/sports-alliance/sports-lib/blob/c1f4c814d8e3605980540dc4b7665a51c7a0c76b/src/events/utilities/activity.utilities.ts#L1052

**Comment by jimmykane** [comment]
Does this look correct? 

<img width="828" alt="Screenshot 2020-11-17 at 19 52 12" src="https://user-images.githubusercontent.com/1268723/99433914-72f6f400-290e-11eb-9dfc-8acaaa38efa0.png">

**Comment by evanodell** [comment]
Yeah that looks correct, and matches with what Suunto itself reports: https://app.suunto.com/move/evanodell/5e76296d33564d7a05716b96/

**Comment by jimmykane** [comment]
Wait the fix is not out 🙏

I was just making sure
